.changeset/small-ducks-heal.md

```diff
@@ -0,0 +1,7 @@
+---
+'@makeswift/runtime': patch
+---
+
+Add `mode` option to `unstable_RichTextV2` control.
+
+Setting the mode of `RichTextV2` to `RichTextMode.Inline` locks down output to only include inline HTML elements. The use case we have in mind is allowing you to double-click edit button text, while protecting you from creating hydration mismatch errors.
```

Comment:
Is  there are type of special indenting that I need to do changesets to format this correctly?